internal/nodes/avi_model_l7_hostname_shard.go

```diff
@@ -292,7 +268,7 @@ func sniNodeHostName(routeIgrObj RouteIngressModel, tlssetting TlsSettings, ingN
 		}
 		SharedHostNameLister().Save(sniHost, ingressHostMap)
 		sniHosts = append(sniHosts, sniHost)
-		shardVsName := DeriveHostNameShardVS(sniHost, key)
+		shardVsName := DeriveShardVS(sniHost, key)
```

Comment:
prefer the older name?

Reply:
I thought of removing the explicit `HostName` usage since there's no other alternative.